code/game/machinery/camera/camera.dm

```diff
@@ -299,6 +299,19 @@ GLOBAL_LIST_EMPTY_TYPED(all_cameras, /obj/structure/machinery/camera)
 		return 1
 	return 0
 
+/obj/structure/machinery/camera/correspondent
+	network = list(CAMERA_NET_CORRESPONDENT)
+	invisibility = INVISIBILITY_ABSTRACT
+	invuln = TRUE
+	unslashable = TRUE
+	unacidable = TRUE
+	colony_camera_mapload = FALSE
+	var/linked_broadcasting
+
+/obj/structure/machinery/camera/correspondent/Initialize(mapload, c_tag_name)
+	c_tag = c_tag_name
+	return ..()
```

Comment:
If you pass the camera item itself here rather than just the tag name, the `linked_broadcasting` variable can be set  inside `Initialize()` rather than separately elsewhere, which is a little bit neater.
```suggestion
	var/linked_broadcasting

/obj/structure/machinery/camera/correspondent/Initialize(mapload, obj/item/device/camera/broadcasting/camera_item)
	c_tag = camera_item.get_broadcast_name()
	linked_broadcasting = camera_item
	return ..()
```